field.go

```diff
@@ -485,6 +486,15 @@ func (f *Field) ToSwaggerParameters(in string) (parameters []Parameter) {
 			parameters = append(parameters, param)
 		}
 	}
```

Comment:
Can you stick it above this paren so it only affects the objects? We don't want it reordering people's arrays.

Reply:
Done.